src/v2/api/subject/SubjectService.ts

```diff
@@ -29,12 +49,17 @@ export class SubjectService {
   }
 
   async get (id: string) {
-    return this.subjectRepository.getSubject(id);
+    const subject = await this.subjectRepository.findById(id);
+    return {
```

Comment:
What is the purpose of this mapping? Can we just `return this.subjectRepository.findById();`